app/ui/design-system/src/lib/Components/InternalToc/index.tsx

```diff
@@ -26,27 +26,22 @@ export function InternalToc({
         On this page
       </div>
       <div className="border-l-1 border-l border-l-gray-100 bg-opacity-80 dark:border-l-gray-800">
-        {/* TODO: Fix: We have to slice otherwise we get the page title */}
-        {headings.length
-          ? headings.slice(1, headings.length).map(({ title, hash }) => {
-              return (
-                <div className="flex" key={Math.random()}>
-                  <a
-                    href={hash}
-                    className={clsx(
-                      "mb-1 cursor-pointer py-2 px-5 text-sm text-primary-gray-400 hover:opacity-75 dark:text-gray-200",
-                      {
-                        "bg-gray-100 bg-opacity-75 font-semibold text-primary-blue dark:bg-primary-gray-dark dark:text-gray-300":
-                          hash === currentHash,
-                      }
-                    )}
-                  >
-                    {title}
-                  </a>
-                </div>
-              )
-            })
-          : "No content"}
+        {headings.map(({ title, hash }, index) => (
+          <div className="flex" key={index}>
+            <a
+              href={hash}
```

Comment:
Should this be scrolling the page to the heading?

Reply:
yeah that's what it does, `hash` here is a url anchor, e.g. `#publickey`

<img width="1612" alt="image" src="https://user-images.githubusercontent.com/921605/175616966-8e7e36cf-8043-432b-a8fd-be6b7684bf0b.png">